internal/extensions.go

```diff
@@ -0,0 +1,63 @@
+package internal
+
+import (
+	"fmt"
+	"github.com/wal-g/wal-g/internal/storages/storage"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"plugin"
+)
+
+var Extensions []Extension
+
+type Extension interface {
+	TryPrintHelp(command, firstArgument string) bool
```

Comment:
Let's improve TryPrintHelp interface, passing here all arguments instead of only first one.

Reply:
Fixed